resources/META-INF/plugin.xml

```diff
@@ -536,6 +536,13 @@
     <annotator language="Dart" implementationClass="io.flutter.editor.FlutterEditorAnnotator"/>
     <errorHandler implementation="io.flutter.FlutterErrorReportSubmitter"/>
 
+    <toolWindow id="Flutter Preview" anchor="right" icon="FlutterIcons.Flutter_13"
```

Comment:
@jwren brings up a good point, that we should iterate on the name (not for this PR). `Flutter preview` as a name may not be obvious to users what it is; others candidates are `Flutter Outline`, `Flutter Structure` (familiar to existing IntelliJ users), and `Flutter Designer`.

We're good with the current name but should keep in the back of our minds that we'll need to settle on what we want the final name to be.